backend/src/main/java/com/bakdata/conquery/util/SimpleObservable.java

```diff
@@ -0,0 +1,51 @@
+package com.bakdata.conquery.util;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class SimpleObservable<T> {
+    public  interface Observer<T> {
+        void onObservableChanged(T data);
+    }
+    // this is the object we will be synchronizing on ("the monitor")
+    private final Object MONITOR = new Object();
+    private Set<Observer<T>> mObservers;
```

Comment:
Kannst du gleich initialisieren
```suggestion
    private final Set<Observer<T>> mObservers = new HashSet<>();
```